osidb/models.py

```diff
@@ -2653,7 +2691,7 @@ def fix_state(self):
         if self.resolution:
             self.resolution = self.resolution.lower()
 
-        # Eg. GITOPS-1472, AAH-682
+            # Eg. GITOPS-1472, AAH-682
```

Comment:
There does not seem to be a reason of this change and it shifts the comment on and incorrect indention